CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/hashmod/HashmodAction.java

```diff
@@ -65,14 +69,23 @@ public void actionPerformed(final ActionEvent ev) {
         final DialogDescriptor dialog = new DialogDescriptor(hashmodPanel, Bundle.MSG_Title(), true, e -> {
```

Comment:
This panel looks something that could be done using plugin parameters and a standalone plugin. This would allow more flexibility in how the plugin can be used, and not lock the UI elements down to the EDT (Swing thread).

Reply:
I was thinking the same thing when I merged in the original version of Hashmod. I was going to make the changes and put a PR up but have not been able to.

As @cygnus-x-1 mentions, there are many advantages to following the plugin framework approach like not having to worry about creating your own GUI for it. Defining the plugin parameters is enough and you can do something like `PluginExecution.withPlugin(PluginRegistry.HASHMOD).interactively(true).executeLater(graph);` and your GUI is done.

Would you be able to modify it @CrucisGamma? Like I said, I was going to give it a go but probably won't be able to do that today.

Reply:
I was essentially copying Add Banner.  Do we modify that one as well?

Reply:
Discussed this with @CrucisGamma and given its currently experimental, I'll get this merged in so @CrucisGamma can continue with testing.

We agreed that creating custom GUIs is not desirable and it turned out @CrucisGamma did try using plugin parameters but ran into some issues.

Happy to revisit this again and look to migrate plugins in the experimental menu over to plugin parameters as a separate task.